src/helper.coffee

```diff
@@ -121,8 +121,16 @@ module.exports = {
     else
       paritybitRef = ((parity % 2) isnt 0)
     return paritybitRef
+	
+  hexChecksum: (data) ->
+    checksum = 0
+    number = @binaryToNumberLSBMSB(data, 0, 31)
```

Comment:
i mean exact this line.

number = @binaryToNumberLSBMSB(data, 0, 31)
Number isnt used anymore